lld/ELF/Symbols.cpp

```diff
@@ -316,12 +316,13 @@ void elf::maybeWarnUnorderableSymbol(const Symbol *sym) {
   if (!config->warnSymbolOrdering)
     return;
 
-  // If UnresolvedPolicy::Ignore is used, no "undefined symbol" error/warning
-  // is emitted. It makes sense to not warn on undefined symbols.
+  // If UnresolvedPolicy::Ignore is used, no "undefined symbol" error/warning is
+  // emitted. It makes sense to not warn on undefined symbols (excluding those
+  // demoted by demoteSymbolsInDiscardedSections).
```

Comment:
This should be just "demoteSymbols".

Reply:
Thanks for the catch. I think both demoteSymbols/demoteLocalSymbolsInDiscardedSections but we probably can just mention demoteSymbols.